6-Typescript/server/tests/it/delete-todo.test.ts

```diff
@@ -1,33 +1,33 @@
-import { TestKit } from '../env/TestKit';
-import {createTodo} from '../env/random'
-import { ITodo } from '../../interfaces/ITodo';
-const testKit = new TestKit()
+import { TestKit } from "../env/TestKit";
+import { createTodo } from "../env/mockData";
+import { ITodo } from "../../interfaces/interfaces";
+const testKit = new TestKit();
 describe("DELETE todo by id /api/todos/:id ", () => {
-testKit.beforeAndAfter();
+    testKit.beforeAndAfter();
 
     describe("given no id of todo in db", () => {
         it("should return 404", async () => {
-            let todo: ITodo = createTodo()            
-            await testKit.mongoDriver.modelPostTodo(todo)
-            testKit.appDriver.setUserId(todo.userId)
-            todo.todoId = 'sdgsdf' 
-            try{
-                const response = await testKit.appDriver.deleteTodoById(todo.todoId) 
-            }
-            catch(err: any) {     
+            let todo: ITodo = createTodo();
+            await testKit.mongoDriver.modelPostTodo(todo);
+            testKit.appDriver.setUserId(todo.userId);
+            todo.todoId = "sdgsdf";
+            try {
+                const response = await testKit.appDriver.deleteTodoById(
+                    todo.todoId
+                );
+            } catch (err: any) {
```

Comment:
any

Reply:
again - this is the error i got when i tried to use a diff type name:
'Catch clause variable type annotation must be 'any' or 'unknown' if specified'